### PR TITLE
UIPFAUTH-67 updated markHighlightedFields releated with UISAUTCOMP-59

### PR DESCRIPTION
**Pull request by przemyslawturek**
Required changes in markHighlightedFields releated with UISAUTCOMP-59

## Issue 
[UIPFAUTH-67](https://issues.folio.org/browse/UIPFAUTH-67)

## Other
Update jest config to use jest-config-stripes
Update jest version and releated to 29.5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,14 +51,15 @@
     "@formatjs/cli": "^4.2.20",
     "@testing-library/jest-dom": "^5.11.1",
     "@testing-library/react": "^11.0.4",
-    "babel-jest": "^26.4.2",
+    "babel-jest": "^29.5.0",
```
Comment (Dmytro-Melnyshyn) [inline package.json:54]: Do tests work without `babel-jest`, `jest`, `jest-environment-jsdom`? All these deps are already listed in `jest-config-stripes`.

**Review by Dmytro-Melnyshyn — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (przemyslawturek) [reply package.json:54]: Unfortunately, tests not working without those deps locally

Comment (BogdanDenis) [reply package.json:54]: Hey @przemyslawturek, are you running tests in a standalone module or as a part of platform? I've just installed this module separately from platform, without jest dependencies and tests are working correctly.
If it's a part of platform - there could be failing tests due to different jest versions in different modules.

Unfortunately test-related dependencies haven't been consistently kept in sync, and removing them from `devDependencies` and relying on `jest-config-stripes` to provide them is going to fix this problem

Comment (BogdanDenis) [reply package.json:54]: Also, could you provide what error you are seeing in tests?

Comment (Dmytro-Melnyshyn) [reply package.json:54]: This may be a local issue, please push a commit without those deps.

**Review by Dmytro-Melnyshyn — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (przemyslawturek) [reply package.json]: Yes it seems this problem is only on standalone version running separately outside platform. Sorry guys for turning you into the wrong path, my bad

**Pull request merged**